.github/workflows/main.yml

```diff
@@ -7,8 +7,8 @@ on:
     tags:
     - v*
   pull_request:
-    branches:
-    - dev
+    branches-ignore:
+    - master
```

Comment:
The `master` will be the branch where we merge most PRs to, right? Is this change correct?

Reply:
You're right, it's wrong.

But I was actually trying to make sure we run the tests for PRs that are based in feature-brances (like this one which is based in `feat/allow-version-override`), with the previous config the tests weren't running for these cases.

So I think we should just remove any branch restriction on the `pull_request` event, what do you think?

Reply:
I think that makes sense. We can always add "[skip ci]" on the commit message if we don't want it to run.